it/controllers/CalculationControllerISpec.scala

```diff
@@ -124,10 +124,15 @@ class CalculationControllerISpec extends ComponentSpecBase {
           elementTextByID("dividend-art-amount")(calculationDataSuccessWithEoYModel.dividends.additionalBand.taxAmount.toCurrencyString),
           elementTextByID("nic2-amount")(calculationDataSuccessWithEoYModel.nic.class2.toCurrencyString),
           elementTextByID("nic4-amount")(calculationDataSuccessWithEoYModel.nic.class4.toCurrencyString),
+          //elementTextByID("gift-aid")(calculationDataSuccessWithEoYModel.giftAid.taxableAmount.toCurrencyString),
           elementTextByID("total-estimate")(calculationDataSuccessWithEoYModel.totalIncomeTaxNicYtd.toCurrencyString),
          isElementVisibleById("eoyEstimate")(expectedValue = true)
 
         )
+
+        res.body.contains("gift-aid") shouldBe false
```

Comment:
Could this be adjusted to be similar to the checks above? Similar to the eoyEstimate one where it is checking that the element "eoyEstimate" is visible, you could have `isElementVisibleById("gift-aid")(expectedValue = false)`